modules/labs-react/layout/lib/Stack.tsx

```diff
@@ -88,19 +72,19 @@ export const HStack = createComponent('div')({
   },
 });
 
-export type VStackProps = Omit<StackProps, 'flexDirection'> & {
+export interface VStackProps extends StackProps {
   /**
    * sets the direction for the stack
    * @default "column"
    * */
   flexDirection?: 'column' | 'column-reverse';
-};
+}
 
 export const VStack = createComponent('div')({
   displayName: 'VStack',
-  Component: ({children, flexDirection = 'column', ...elemProps}: VStackProps, ref, Element) => {
+  Component: ({children, ...elemProps}: VStackProps, ref, Element) => {
     return (
-      <Stack as={Element} ref={ref} flexDirection={flexDirection} {...elemProps}>
+      <Stack as={Element} ref={ref} flexDirection="column" {...elemProps}>
```

Comment:
Changes to this file are optional. The changes simplify `Stack` by removing unnecessary Typescript and decreasing prop name duplication.